components/tabs/index.jsx

```diff
@@ -38,7 +38,7 @@ import EventUtil from '../../utilities/event';
 function isTabNode(node) {
 	return (
 		(node.nodeName === 'A' && node.getAttribute('role') === 'tab') ||
-		(node.nodeName === 'LI' && node.getAttribute('role') === 'tab')
+		(node.nodeName === 'LI' && node.getAttribute('role') === 'presentation')
```

Comment:
This is a very broad check. Would be better to check for classes. 
This check is true for some of our content within a tab. If I click on node at position N within the content, the tab at position N becomes active.

Reply:
Good catch. `tab` is what we were looking but `presentation` gets used often in SLDS.